src/main/java/dev/dfonline/codeclient/config/Config.java

```diff
@@ -81,6 +81,11 @@ public class Config {
     public CPUDisplayCorner CPUDisplayCornerOption = CPUDisplayCorner.TOP_LEFT;
     public boolean HideScopeChangeMessages = true;
 
+    public boolean HighlighterEnabled = true; // whether expression highlighting is enabled.
+    public boolean HighlightExpressions = true; // if the highlighter marks up df expressions
+    public boolean HighlightMiniMessage = true; // if the highlighter marks up minimessage.
+    public int MiniMessageTagColor = 0x808080; // the color of minimessage tags in the edit box.
```

Comment:
nothing else has comments. for consistency remove them or we could move to adding javadoc to each one, although they do have their description under getLibConfig